cloud/deployment/deployment.go

```diff
@@ -1172,6 +1180,98 @@ func Delete(deploymentID, ws, deploymentName string, forceDelete bool, platformC
 	return nil
 }
 
+func UpdateDeploymentHibernationOverride(deploymentID, ws, deploymentName string, isHibernating bool, overrideUntil *time.Time, force bool, platformCoreClient astroplatformcore.CoreClient) error {
+	// Set wording based on the hibernation action
+	var action string
+	if isHibernating {
+		action = "hibernate"
+	} else {
+		action = "wake up"
+	}
+
+	// get deployment
+	currentDeployment, err := GetDeployment(ws, deploymentID, deploymentName, true, isDevelopmentDeployment, platformCoreClient, nil)
+	if err != nil {
+		return err
+	}
+	if currentDeployment.Id == "" {
+		fmt.Printf("No Deployments found in workspace %s to %s\n", ansi.Bold(ws), action)
+		return nil
+	}
+
+	if currentDeployment.IsDevelopmentMode == nil || !*currentDeployment.IsDevelopmentMode {
+		return ErrNotADevelopmentDeployment
+	}
+
+	// prompt user
+	if !force {
+		i, _ := input.Confirm(
+			fmt.Sprintf("\nAre you sure you want to override to %s for %s Deployment?", ansi.Bold(action), ansi.Bold(currentDeployment.Name)))
+
+		if !i {
+			fmt.Printf("\nCanceling %s override", action)
+			return nil
+		}
+	}
+
+	overrideDeploymentHibernationBody := astroplatformcore.OverrideDeploymentHibernationBody{
+		IsHibernating: isHibernating,
+		OverrideUntil: overrideUntil,
+	}
+
+	deploymentHibernationOverride, err := CoreUpdateDeploymentHibernationOverride(currentDeployment.OrganizationId, currentDeployment.Id, overrideDeploymentHibernationBody, platformCoreClient)
+	if err != nil {
+		return err
+	}
+
+	if deploymentHibernationOverride.OverrideUntil != nil {
+		fmt.Printf("\nSuccessfully overrode to %s until %s\n", ansi.Bold(action), ansi.Bold(deploymentHibernationOverride.OverrideUntil.Format(time.RFC3339)))
+		fmt.Printf("Schedule will resume in %s.\n", ansi.Bold(time.Until(*deploymentHibernationOverride.OverrideUntil).Round(time.Second).String()))
+	} else {
+		fmt.Printf("\nSuccessfully overrode to %s until further notice\n", ansi.Bold(action))
+		fmt.Println("Schedule will not resume until override is removed.")
+	}
+
+	return nil
+}
+
+func DeleteDeploymentHibernationOverride(deploymentID, ws, deploymentName string, force bool, platformCoreClient astroplatformcore.CoreClient) error {
+	// get deployment
+	currentDeployment, err := GetDeployment(ws, deploymentID, deploymentName, true, isDevelopmentDeployment, platformCoreClient, nil)
+	if err != nil {
+		return err
+	}
+	if currentDeployment.Id == "" {
+		fmt.Printf("No Deployments found in workspace %s to remove the hibernation override for\n", ansi.Bold(ws))
```

Comment:
```suggestion
		fmt.Printf("No Deployments with a hibernation override that can be removed found in Workspace %s\n", ansi.Bold(ws))
```

Does this make more sense?